sales/send_quotations/get_paid_to_validate.rst

```diff
@@ -2,42 +2,56 @@
 Get paid to confirm an order
 ============================
 
-You can use online payments to get orders automatically confirmed.
-Saving the time of both your customers and yourself.
+In general, online payments are considered as a fast and secure alternative to traditional payment
+methods. It is generally cheaper, easier, faster and highly efficient for international transactions
+than other payment methods. With Odoo Sales, you can use online payments to get automatic order
+confirmation. Online payments are made instantly, so it's very convenient and saves lots of time
+in a basic sales process.
 
-Activate online payment
-=======================
-
-Go to :menuselection:`SALES --> Configuration --> Settings` and activate
-the *Online Signature & Payment* feature.
+How to enable online payment
+============================
 
-.. image:: media/getpaidtovalidate01.png
-   :align: center 
+Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Online Payment**
+feature.
 
-Once in the *Payment Acquirers* menu you can select and configure your
-acquirers of choice.
+.. image:: media/get_paid_1.png
+   :align: center
+   :class: img-thumbnail
+   :alt: How to enable online payment on Odoo Sales?
 
-You can find various documentation about how to be paid with payment
-acquirers such as `Paypal
-<../../ecommerce/shopper_experience/paypal>`_,
+There, you will have direct access to the **Payment Acquirers** page. It will allow you to select
+and configure your acquirers of choice. Before creating or modifying a payment acquirer, be sure to
+check out our documentation about how to be paid with payment acquirers such as `Paypal
+<https://www.odoo.com/documentation/user/13.0/ecommerce/shopper_experience/paypal.html>`_,
 `Authorize.Net (pay by credit card)
-<../../ecommerce/shopper_experience/authorize>`_, 
-and others under the `eCommerce documentation
-<../../ecommerce>`_.
-
-
+<https://www.odoo.com/documentation/user/13.0/ecommerce/shopper_experience/authorize.html>`_,
+and others in the `eCommerce documentation
+<https://www.odoo.com/documentation/user/13.0/ecommerce.html>`_.
 
 .. note::
- If you are using `quotation templates 
- <../quote_template>`_, 
- you can also pick a default setting for each template.
+   If you are familiar with this documentation: `Use quotation templates <../quote_template>`_, you
```

Comment:
There is two things going on here:
1. The original link was broken because it was not using a `doc` directive. It would have been possible (but not recommended) to use a simple hyperlink reference (with a leading `_`) but then the ".html" would have been required in the relative url.
2. The reason for which a label "quotation templates" was being used is that the link was inlined in a sentence and we didn't want the full page title to be rendered. Now that you want the page title to display in full, it's better to simply remove the customer label and let the `doc` directive generate it by its own.